.github/workflows/continuous-integration.yml

```diff
@@ -81,7 +78,7 @@ jobs:
 
       - name: "Install PHP with PCOV"
         uses: "shivammathur/setup-php@v2"
-        if: "${{ matrix.php-version != '7.1' }}"
+        if: "${{ matrix.php-version != '7.4' }}"
```

Comment:
This change looks wrong to me. We should keep using PCov on PHP 7.4 (removing the PHP 7.1 logic entirely)